Automattic/woocommerce-payments
# Add bnpl logos to thank you page.

Fixes #8249

#### Changes proposed in this Pull Request
This PR adds the bnpl payment method logo to the thank you page. I'm utilizing the `get_theme_icon()` to get the light/dark icon based on whichever checkout upe transient has been set (classic or block checkout). WooPay also adds a logo in this spot so I borrowed the approach used there but then generalized the wrapper class so I didn't have to repeat CSS. I also added a WooPay-specific class to that wrapper as well as the new bnpl wrappers in case we ever need to scope CSS to them individually. This implementation also adds a `wc_payments_thank_you_page_bnpl_payment_method_logo_url` filter to allow users to change the logo URL in case our transient specifies the dark logo but a light logo works better on the thank you page. It also falls back to the payment method title (how it currently works) if it can't get the logo URL for some reason.


#### Testing instructions
* Complete a checkout with a BNPL method and you should see the logo on the thank you page instead of the plain title.
* Please check multiple themes with both light and dark versions to ensure that the proper logo style is pulled. You'll need to visit the checkout page after the theme change to make sure that the UPE appearances are generated and saved.

-------------------


**Post merge**
